src/app/redux/actions.ts

```diff
@@ -1,12 +1,16 @@
 import { UPDATE_SELECTED_STOCK } from "../redux/constants";
-import { StockProps } from "../../features//headline/components/Headline";
 import { ActionWithPayload } from "store/utilities";
 
+export type Stock = {
```

Comment:
tip: just as an idea across your app, if you're finding yourself exporting a lot of different types it might be worth creating a 'models' folder to store them in. You'll probably find that you'll have models at a component level as well as a global level.

Reply:
Good point, will think about it for a new PR